evm-contracts/src/v0.7/dev/Operator.sol

```diff
@@ -281,6 +281,57 @@ contract Operator is
     s_forwarder.setAuthorizedSenders(forwarderSenders);
   }
 
+  /**
+   * @notice If the s_forwarder is owned by a different address, deploy and set a new one
+   */
+  function deployForwarder()
+    external
+    override
+    onlyOwner()
+    returns (
+      address
```

Comment:
We can do this, but because this overrides an interface definition it opens a can of worms where the `OperatorForwarder` needs an `OperatorForwarderInterface` and the `OperatorForwarderInterface` subsequently needs a `ConfirmedOwnerInterface`.